### DISCUSSION TITLE
When using Transparency, if nvim-tree is not in focus, the status bar gets blacked out

### DISCUSSION THREAD

**Discussion by OWL4C** [nvim-tree/nvim-tree.lua#2763] [General]
If Nvim Tree is the active panel
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/124401812/e514a325-10f8-4836-ac82-2a2aa94c7bb6)
If i switch towards another panel 
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/124401812/694c7ee3-bad1-4c91-8a08-ce278c8f6565)
Do i have something misconfigured?

**Comment by alex-courtis** [comment]
When not focused, nvim-tree will use `NvimTreeStatusLineNC`. That defaults to the standard `StatusLineNC`.

Please see `:help nvim-tree-highlight` for details on all the highlight groups and their defaults.

Run `:NvimTreeHiTest` to see the nvim-tree and nvim current highlights, their links and concrete values.
